server/src/main/scala/services/exports/flights/templates/CedatFlightsExport.scala

```diff
@@ -68,7 +68,7 @@ case class CedatFlightsExport(start: SDateLike, end: SDateLike, terminal: Termin
       fws.apiFlight,
       millisToDateStringFn,
       millisToTimeStringFn
-    ) ++ List(fws.apiFlight.bestPcpPaxEstimate.pax.toString) ++ splitsForSources
+    ) ++ List(fws.apiFlight.bestPcpPaxEstimate.pax.getOrElse("").toString) ++ splitsForSources
```

Comment:
We get a warning here due to the different types, ie `pax` is `Option[Int]`, and we're saying `getOrElse("")`
I think the fix is `pax.map(_.toString).getOrElse("")`